src/components/Article/Article.jsx

```diff
@@ -3,3 +3,69 @@
 // Create an Article function returning the HTML of article block
 
 // Add a default export statement for Article component to use it in the other files
+import './Article.css';
+
+export const Article = () => {
```

Comment:
Not fixed, you need to add default export to your components 
<img width="715" alt="image" src="https://github.com/user-attachments/assets/dd7a2e58-2c75-4e1a-87cc-23986657af74" />
```suggestion
// Add a default export statement for Article component to use it in the other files
import './Article.css';

const Article = () => {
```